Source/Core/AudioCommon/AudioCommon.cpp

```diff
@@ -76,13 +69,12 @@ void ShutdownSoundStream()
 {
   INFO_LOG(AUDIO, "Shutting down sound stream");
 
+  if (SConfig::GetInstance().m_DumpAudio && s_audio_dump_start)
+    StopAudioDump();
+
   if (g_sound_stream)
```

Comment:
Is it even possible for `g_sound_stream` to not be initialized at this point?

(Even so, `StopAudioDump` depends on `g_sound_stream` being non-`nullptr`.)

Reply:
I don't think it'll ever happen, no, though I guess someone could call Shutdown multiple times...

I didn't notice the dependency inside `StopAudioDump`! I've added a check for a non-null soundstream to that function. I've also removed this conditional block, since `SetSoundStreamRunning` checks it anyways, and `unique_ptr::reset` doesn't care.